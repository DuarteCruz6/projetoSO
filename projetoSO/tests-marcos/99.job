WRITE [(k8,29)(i5,39)(q3,72)(i7,41)(j8,37)]
DELETE [i7,k8,j8,i5]
SHOW
READ [q3]
READ [q3]
WRITE [(c4,7)(o0,85)(i8,46)(p5,82)(d0,63)]
READ [c4]
WRITE [(m1,28)(v4,11)(p0,31)]
DELETE [o0,d0,v4]
READ [q3]
WRITE [(e6,91)(p2,29)]
READ [m1]
WRITE [(k5,83)(q5,65)(d1,93)(n9,77)]
DELETE [i8,n9,q3,p2,e6]
WRITE [(z5,17)]
WRITE [(c3,28)]
WRITE [(m9,60)(m0,61)]
READ [q5]
WRITE [(p9,14)]
READ [c4]
WRITE [(u6,22)(d4,57)(t3,27)]
WRITE [(q6,34)(w1,22)(g9,49)]
DELETE [d4,k5,q5,q6,p5]
WRITE [(o9,59)(c0,12)(j5,11)]
WRITE [(t6,16)]
WRITE [(k6,25)]
WRITE [(e2,74)(z4,11)(v7,33)(l2,44)(o4,16)]
DELETE [p0,g9,d1,c4,o4]
WRITE [(t0,67)]
WRITE [(f3,90)(j6,26)]
READ [m1]
WRITE [(l9,6)]
READ [z5]
DELETE [o9,m9,l2,t6,z5]
WRITE [(k7,75)(j7,46)]
DELETE [e2,m1,j6,k7,m0]
WRITE [(n5,15)(k4,2)]
WAIT 2000
WRITE [(p6,79)]
READ [j7]
WRITE [(p4,19)]
DELETE [k4]
DELETE [j5,l9,u6,t3]
READ [c0]
READ [p6]
DELETE [n5,z4,c0,p4,v7]
WRITE [(w6,39)(z7,56)(h3,50)(f9,68)(i9,60)]
WRITE [(q7,30)]
READ [z7]
DELETE [f9,k6]
WRITE [(c1,90)(p3,4)(q1,66)(w9,86)(n3,9)]
WRITE [(k6,59)(w5,73)(n4,16)(k0,16)(n8,26)]
WRITE [(u0,60)]
WRITE [(o2,47)(y8,68)(m3,6)(r5,43)(w8,76)]
BACKUP
