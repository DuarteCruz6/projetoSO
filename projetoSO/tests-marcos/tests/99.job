WRITE [(f1,97)(w5,14)(b2,81)(k1,99)(k8,5)]
WRITE [(l9,40)(n5,65)(v9,49)]
WRITE [(k5,26)]
WRITE [(x1,6)(t9,52)]
WRITE [(z6,33)(u5,85)(j4,65)(f4,36)(i3,5)]
WRITE [(g4,89)(s8,34)(e4,64)]
SHOW
WRITE [(p1,33)(v4,4)]
SHOW
WRITE [(m8,60)(s3,5)(q2,1)(o0,30)(o3,17)]
WRITE [(i9,77)(c0,98)(y6,96)]
DELETE [i9,w5]
WRITE [(j2,85)]
DELETE [x1,k1,s3,u5,j4]
WRITE [(r7,97)(f9,74)(l0,94)]
WRITE [(k4,97)(x3,96)]
WRITE [(u7,90)]
WRITE [(d6,34)(b8,7)(w1,80)(q3,54)(u6,9)]
READ [q2]
DELETE [u7]
BACKUP
WAIT 2000
DELETE [w1,n5,q2,e4,o0]
WRITE [(y3,7)(z0,87)]
DELETE [m8,q3]
WRITE [(w8,16)(b3,55)]
WRITE [(h1,0)(g6,94)(i8,35)]
WRITE [(g0,58)(x2,92)(n0,78)]
WRITE [(d4,29)(p9,30)(r5,58)(l8,26)(a3,89)]
WRITE [(r4,18)]
WRITE [(d8,16)(o9,20)(r1,21)(o5,38)(r2,94)]
READ [h1]
DELETE [n0,o3,o9]
READ [r1]
WAIT 3000
WRITE [(v8,64)(n0,2)]
