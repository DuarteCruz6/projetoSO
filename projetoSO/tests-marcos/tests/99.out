(b2, 81)
(e4, 64)
(f4, 36)
(f1, 97)
(g4, 89)
(i3, 5)
(j4, 65)
(k5, 26)
(k8, 5)
(k1, 99)
(l9, 40)
(n5, 65)
(s8, 34)
(t9, 52)
(u5, 85)
(v9, 49)
(w5, 14)
(x1, 6)
(z6, 33)
(b2, 81)
(e4, 64)
(f4, 36)
(f1, 97)
(g4, 89)
(i3, 5)
(j4, 65)
(k5, 26)
(k8, 5)
(k1, 99)
(l9, 40)
(n5, 65)
(p1, 33)
(s8, 34)
(t9, 52)
(u5, 85)
(v4, 4)
(v9, 49)
(w5, 14)
(x1, 6)
(z6, 33)
[(q2,1)]
[(h1,0)]
[(r1,21)]
